docs/src/modules/components/ThemeContext.js

```diff
@@ -244,6 +244,7 @@ export function ThemeProvider(props) {
   }, [theme]);
 
   useEnhancedEffect(() => {
+    // track usage of dark mode in google analytics
```

Comment:
This would be more accurate. It's not about GA.

```suggestion
    // To support light and dark mode images in the docs
```

But if we have `<html data-mui-color-scheme="">` enforced, I think that we can replace that logic, maybe in a follow-up to keep this PR simpler.

Reply:
That's what I think. Will add a TODO comment.

Reply:
> This would be more accurate. It's not about GA.

Are you sure because it comes from https://github.com/mui/material-ui/pull/29419?

Reply:
💯% sure, the code was moved